webextensions/api/webRequest.json

```diff
@@ -117,18 +117,29 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/RequestFilter",
             "support": {
               "chrome": {
+                "notes": "If a filter contains unrecognized values in its <code>types</code> property, this is treated as an error.",
                 "version_added": true
               },
               "edge": {
+                "notes": "If a filter contains unrecognized values in its <code>types</code> property, this is treated as an error.",
                 "version_added": "14"
               },
-              "firefox": {
-                "version_added": "45"
-              },
+              "firefox": [
+                {
+                  "version_added": "45"
```

Comment:
Yeah, structuring this is a bit challenging. In this case, because there's no real standard and Firefox's new behavior seems to have gotten more permissive, I'm inclined to stuff this into notes. Maybe two notes, like this:

1. `From Firefox 78, unrecognized <code>types</code> values are discarded with a warning.`
1. `Before Firefox 78, if a filter contains unrecognized values…`